lib/assertions.ts

```diff
@@ -492,15 +492,42 @@ export default class DOMAssertions {
    * @see {@link #hasClass}
    */
   hasStyle(expected: object, message?: string): void {
+    this.hasPseudoElementStyle(null, expected, message);
+  }
+
+
+  /**
+   * Assert that the pseudo element for `selector` of the [HTMLElement][] has the `expected` style declarations using
+   * [`window.getComputedStyle`](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle).
+   *
+   * @name hasPseudoElementStyle
+   * @param {string} selector
+   * @param {object} expected
+   * @param {string?} message
+   *
+   * @example
+   * assert.dom('.progress-bar').hasPseudoElementStyle(':after', {
+   *   content: '";"',
+   * });
+   *
+   * @see {@link #hasClass}
+   */
```

Comment:
can you check that `yarn docs` reads the comment correctly? the `API.md` change should then also be committed, I guess

Reply:
It does not work correctly.